buildSrc/build.gradle.kts

```diff
@@ -0,0 +1,14 @@
+plugins {
+    `kotlin-dsl`
+}
+
+repositories {
+    google()
+    jcenter()
+}
+
+dependencies {
+    implementation("com.moowork.gradle:gradle-node-plugin:1.3.1")
+    implementation("org.jetbrains.kotlin:kotlin-gradle-plugin:1.3.31")
```

Comment:
let's use `ext.kotlin_version` variable?

Reply:
Extracted to `Versions.kotlin`, ext `build.gradle` is not available in `buildSrc`.